kcidb/test_oo.py

```diff
@@ -253,6 +263,16 @@ def assertContains(self, container, contents):
                     obj, contents[0],
                     msg=f"Invalid type of {contents[0]} #{i}")
 
+        elif isinstance(contents, list):
+            self.assertItemIsTwo(contents)
+            self.assertIsInstance(container, contents[0])
+            self.assertEqual(container.parent, contents[1])
+            self.assertContains(container.tests, (Test, 4))
+
+            # Check that each tests have the same path.
+            self.assertEqual(len({obj.path for obj in container.tests}), 1)
+            self.assertEqual(container.nodes, {})
+
```

Comment:
This is a very specific change for such a generic function, and it's not documented. Did it end up here by accident? Could you explain what you're trying to do here?

Reply:
I was trying to use `assertContain` to cater to `Node` conditions. I shouldn't have done it that way.

Reply:
```python
self.assertEqual(len({obj.path for obj in container.tests}), 1)
```
? 
I was checking that the `Test` returned by each sub-nodes test property has the same path.